internal/deno.json

```diff
@@ -9,6 +9,7 @@
     "./diff": "./diff.ts",
     "./format": "./format.ts",
     "./styles": "./styles.ts",
-    "./types": "./types.ts"
+    "./types": "./types.ts",
+    "./support": "./support.ts"
```

Comment:
While mostly used internally, the public endpoint here is created for use by other public APIs.

Reply:
Sounds like recommending other libs to use this endpoint? I'm not in favor of that policy